Advanced Featuretools RUL.ipynb

```diff
@@ -1180,38 +1180,361 @@
+      "  FutureWarning)\n",
+      " 47%|████▋     | 14/30 [03:16<03:43, 13.97s/it]/Users/nate.parsons/dev/featuretools-demos/predict-remaining-useful-life/env/lib/python3.7/site-packages/sklearn/utils/validation.py:71: FutureWarning: Pass n_features_to_select=25 as keyword args. From version 0.25 passing these as positional arguments will result in an error\n",
+      "  FutureWarning)\n",
+      "/Users/nate.parsons/dev/featuretools-demos/predict-remaining-useful-life/env/lib/python3.7/site-packages/sklearn/utils/validation.py:71: FutureWarning: Pass n_features_to_select=25 as keyword args. From version 0.25 passing these as positional arguments will result in an error\n",
+      "  FutureWarning)\n",
+      "/Users/nate.parsons/dev/featuretools-demos/predict-remaining-useful-life/env/lib/python3.7/site-packages/sklearn/utils/validation.py:71: FutureWarning: Pass n_features_to_select=25 as keyword args. From version 0.25 passing these as positional arguments will result in an error\n",
+      "  FutureWarning)\n",
+      "/Users/nate.parsons/dev/featuretools-demos/predict-remaining-useful-life/env/lib/python3.7/site-packages/sklearn/utils/validation.py:71: FutureWarning: Pass n_features_to_select=25 as keyword args. From version 0.25 passing these as positional arguments will result in an error\n",
+      "  FutureWarning)\n",
+      "/Users/nate.parsons/dev/featuretools-demos/predict-remaining-useful-life/env/lib/python3.7/site-packages/sklearn/utils/validation.py:71: FutureWarning: Pass n_features_to_select=25 as keyword args. From version 0.25 passing these as positional arguments will result in an error\n",
+      "  FutureWarning)\n",
+      " 50%|█████     | 15/30 [03:30<03:30, 14.05s/it]/Users/nate.parsons/dev/featuretools-demos/predict-remaining-useful-life/env/lib/python3.7/site-packages/sklearn/utils/validation.py:71: FutureWarning: Pass n_features_to_select=23 as keyword args. From version 0.25 passing these as positional arguments will result in an error\n",
+      "  FutureWarning)\n",
+      "/Users/nate.parsons/dev/featuretools-demos/predict-remaining-useful-life/env/lib/python3.7/site-packages/sklearn/utils/validation.py:71: FutureWarning: Pass n_features_to_select=23 as keyword args. From version 0.25 passing these as positional arguments will result in an error\n",
+      "  FutureWarning)\n",
+      "/Users/nate.parsons/dev/featuretools-demos/predict-remaining-useful-life/env/lib/python3.7/site-packages/sklearn/utils/validation.py:71: FutureWarning: Pass n_features_to_select=23 as keyword args. From version 0.25 passing these as positional arguments will result in an error\n",
+      "  FutureWarning)\n",
+      "/Users/nate.parsons/dev/featuretools-demos/predict-remaining-useful-life/env/lib/python3.7/site-packages/sklearn/utils/validation.py:71: FutureWarning: Pass n_features_to_select=23 as keyword args. From version 0.25 passing these as positional arguments will result in an error\n",
```

Comment:
Is there something we can do about these future warnings?

Reply:
I'm not sure exactly - I think it might require an update to the BTB package, but I wasn't super clear where they were coming from. I'll investigate further.